### ISSUE TITLE
fix wikis tests [development_fixes2]

### ISSUE THREAD

**Issue by Dinaamagdy** [threefoldtech/jumpscaleX_threebot#309]
need to fix wikis tests (https://github.com/threefoldtech/jumpscaleX_threebot/blob/development/ThreeBotPackages/zerobot/webinterface/tests/test_wiki.py)

the tests should follow the changes of the wikis structure 
https://github.com/threefoldtech/jumpscaleX_threebot/tree/development/ThreeBotPackages/zerobot/webinterface/wiki/wikis

**Comment by Dinaamagdy** [comment]
https://github.com/threefoldtech/jumpscaleX_threebot/issues/310

**Comment by AhmedHanafy725** [comment]
wiki tests are fixed here 989bd9ef9045fadd8825d51284fda1a6e0b1edd1 and https://github.com/threefoldtech/jumpscaleX_core/commit/7ddc3254903091d1272f4235ce03364d8cdad7ec.
And they are running on zeroCI

![image](https://user-images.githubusercontent.com/41957921/71446786-d644ca00-272f-11ea-89e5-d2df1e14bc55.png)

Only the last test is failing as the fix on `jumpscaleX_core` hasn't merged yet on `development` branch

#### Verification steps:
```bash
nosetests-3.4 -v /sandbox/code/github/threefoldtech/jumpscaleX_threebot/ThreeBotPackages/zerobot/webinterface/tests/test_wiki.py
```

**Comment by rkhamis** [comment]
Verified
